master/pkg/tasks/task_command.go

```diff
@@ -21,8 +21,9 @@ type GenericCommandSpec struct {
 
 	Keys *ssh.PrivateAndPublicKeys
 
-	Port     *int
-	ProxyTCP bool
+	Port        *int
+	ProxyTCP    bool
+	UseHTTPAuth bool
```

Comment:
Perhaps it should be inverted into `AllowUnauthenticated` or something so that the default is secure.